interface/billing/search_payments.php

```diff
@@ -219,98 +219,99 @@
 <?php include_once("{$GLOBALS['srcdir']}/ajax/payment_ajax_jav.inc.php"); ?>
 
 <script>
-function refreshSearch() {
```

Comment:
The search button here is off in terms of padding/margins. Also the navbar is not correct.